CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 1.3.0
+### Login SSO Documentation
+
+-  Fix a bug in the Login SSO sample code
```

Comment:
@jorgemasta Can you add a bit more specificity here please? Since these change logs get referenced quite a bit, it will help in that regard + allow developers to better understand from a high level overview what was fixed. Thanks!

Reply:
Done 👍